README.md

```diff
@@ -90,20 +103,108 @@ Note that if a new sample is added to the cohort, it's only necessary to perform
 the `extract` step on that sample and then run the (fast) `relate` step again with all
 of the extracted files.
 
-## VCF
+## Extended Usage
+For each command of somalier, extended parameters are listed in `--help` of each subcommand.
+```
+$./somalier --help
+Commands:
+  extract      :   extract genotype-like information for a single sample from VCF/BAM/CRAM.
+  relate       :   aggregate `extract`ed information and calculate relatedness among samples.
+  ancestry     :   perform ancestry prediction on a set of samples, given a set of labeled samples
+  find-sites   :   create a new sites.vcf.gz file from a population VCF (this is rarely needed).
+```
+### somalier extract
+```
+$somalier extract --help
+
+extract genotype-like information for a single-sample at selected sites
 
-`somalier` can `extract` from a multi or single-sample VCF or a GVCF. This will be much faster, in cases where it's available,
-this would look like:
+Usage:
+  somalier extract [options] sample_file
 
+Arguments:
+  sample_file      single-sample CRAM/BAM/GVCF file or multi/single-sample VCF from which to extract
+
+Options:
+  -s, --sites=SITES          sites vcf file of variants to extract
+  -f, --fasta=FASTA          path to reference fasta file
+  -d, --out-dir=OUT_DIR      path to output directory (default: .)
+  --sample-prefix=SAMPLE_PREFIX
+                             prefix for the sample name stored inside the digest
 ```
-somalier extract -d extracted/ --sites sites.vcf.gz -f /data/human/g1k_v37_decoy.fa joint.vcf.gz
+### somalier relate
+```
+$somalier relate --help
+
+calculate relatedness among samples from extracted, genotype-like information
+
+Usage:
+  somalier relate [options] [extracted ...]
+
+Arguments:
+  [extracted ...]  $sample.somalier files for each sample. the first 10 are tested as a glob patterns
+
+Options:
+  -g, --groups=GROUPS        optional path  to expected groups of samples (e.g. tumor normal pairs).
+specified as comma-separated groups per line e.g.:
+    normal1,tumor1a,tumor1b
+    normal2,tumor2a
+  --sample-prefix=SAMPLE_PREFIX
+                             optional sample prefixes that can be removed to find identical samples. e.g. batch1-sampleA batch2-sampleA
+  -p, --ped=PED              optional path to a ped/fam file indicating the expected relationships among samples.
+  -d, --min-depth=MIN_DEPTH  only genotype sites with at least this depth. (default: 7)
+  --min-ab=MIN_AB            hets sites must be between min-ab and 1 - min_ab. set this to 0.2 for RNA-Seq data (default: 0.3)
+  -u, --unknown              set unknown genotypes to hom-ref. it is often preferable to use this with VCF samples that were not jointly called
+  -i, --infer                infer relationships (https://github.com/brentp/somalier/wiki/pedigree-inference)
+  -o, --output-prefix=OUTPUT_PREFIX
+                             output prefix for results. (default: somalier)
+```
+### somalier ancestry (somalier pca)
```

Comment:
help message for `somalier ancestry` and `somalier find-sites` are removed in the new commit 57c89de.